lib/artist.rb

```diff
@@ -0,0 +1,39 @@
+class Artist
+  extend Concerns::Findable
+
+  attr_accessor :name
+  attr_accessor :songs
+
+  @@all = []
+
+  def initialize(name)
+    @name = name
```

Comment:
#### Code Quality - Critical
```initialize(name)``` is common for ```Song``` ```Artist``` and ```Genre``` and should therefore be abstracted into a parent class to avoid repetition.